source/nodejs/adaptivecards-controls/src/popupmenu.ts

```diff
@@ -100,7 +100,6 @@ export class PopupMenu extends PopupControl {
             this._overlayElement = document.createElement("div");
             this._overlayElement.className = "ms-ctrl-overlay";
             this._overlayElement.tabIndex = 0;
-            this._overlayElement.style.zIndex = "1000";
```

Comment:
is this related to teh templating changes or bad merge?

Reply:
The popup menu control was too aggressive in wanting to be in the foreground, and interfered with the designer.